Dockerfile.test

```diff
@@ -1,18 +1,37 @@
-FROM fedora:28
-
-RUN dnf -y install make libgit2-glib tito python3-pylint  \
-                    python3-nose python3-mako python3-flask \
-                    python3-coverage libselinux-python3 sudo \
-                    pykickstart python3-pytoml python3-sphinx \
-                    python3-semantic_version python3-rpmfluff \
-                    anaconda-tui python3-gevent beakerlib && \
-    useradd weldr
-
-RUN mkdir /lorax
-COPY . /lorax
-# remove byte-compiled files to avoid issues between Python 2/3
-# this can happen when you switch between rhel7 and master branches
-RUN find /lorax -name "*.pyc" -exec rm -f {} \;
-
-WORKDIR /lorax
-RUN make test
+FROM registry.access.redhat.com/rhel8-beta
```

Comment:
Is this expected to work outside the internal lan? If not, we need to stick with f28 until centos8 is released.

Reply:
docker pull works. From what I can tell these images are publicly available but they don't have any subscriptions enabled. Fedora 28 sounds like an easier workaround. I will probably go ahead with that and open a task to not forget to update the image at a later point.

Reply:
Switched to Fedora 28 for now.